Server/BackEndMonitoringServer/src/CService.cpp

```diff
@@ -0,0 +1,149 @@
+#include "stdafx.h"
+
+#include "CService.h"
+#include "CThreadPool.h"
+#include "Clogger/include/Log.h"
+
+// We need this to test the service
+#define DEFAULT_BUFLEN 512
+#define DEFAULT_PORT "27015"
+
+void CService::RunServer()
+{
+    std::fstream stream("Log.txt", std::ios_base::out);
+    CLogBuilder builder("Logger", ELogLevel::DEBUG_LEVEL);
+    builder.AddThreadUnsafeStream(stream).SetLogConfig(ELogConfig::CALL_TIME,
+        ELogConfig::FILE_NAME, ELogConfig::FUNCTION_NAME,
+        ELogConfig::LINE_NUMBER, ELogConfig::MESSAGE, ELogConfig::PARAMS);
+    auto logger = builder.BuildSharedLog();
+
+    size_t num_threads = 20;
+    int port = 1111;
+    std::string ip_address = "127.0.0.1";
```

Comment:
maybe better to write comment that here will be XMLParser used?

Reply:
Added \\Todo